web-client/src/components/containers/ImageCard.vue

```diff
@@ -20,7 +20,7 @@
       <CustomButton
         class="text-xs"
       >
-        Click to show on Maps
+        click to show on maps
```

Comment:
I feel we need a convention for capitalization, so far I think the web-app has been capitalizing everything and I gather you'd go for all lower case, what about only capitalizing the first word? Right now the web-app is not consistent, with some text capitalized and other in lower case

Reply:
I realized that maybe it's because some text is stylized in uppercase, is it to removing casing in the code of text being stylized in upper case? I pushed a commit to revert the lowercasing of the last 2 training steps (because the first 3 were capitalized and not being transformed to uppercase)

Reply:
> I realized that maybe it's because some text is stylized in uppercase, is it to removing casing in the code of text being stylized in upper case?

correct! having a specific case (lowercase being my default) was looking weird when it was actually uppercased on display. and matched with `ignoreCase` throughout the tests. so having everyone of theses fields as the same casing made it clearer.

> I pushed a commit to revert the lowercasing of the last 2 training steps which are not uppercased in the styling (the first 3 were still in capital casing)

good catch, I was a bit fast on it. it's quite probable that I forgot some other mixed-case-value-that-are-shown-as-uppercase field, but that's not that important.